src/prisma/schema.prisma

```diff
@@ -11,7 +11,25 @@ generator client {
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
-  email    String @unique
+  id       Int      @id @default(autoincrement())
+  email    String   @unique
   password String
+  groups   Member[]
+}
```

Comment:
timestamps are good practice ..  createdAt,  updatedAt .. but always depends on what you have in your DB, and the business and how cost the change is .. if you have partition or shared .. maybe the alter table for these columns doesn't make sense.